voltixApp/VoltixApp/Views/Keysign/ParticipantDiscovery.swift

```diff
@@ -33,6 +33,9 @@ class ParticipantDiscovery: ObservableObject {
                             let peers = try decoder.decode([String].self, from: data)
                             DispatchQueue.main.async {
                                 for peer in peers {
```

Comment:
```for peer in peers where peer != localParty {```

Reply:
I believe it is faster without filtering since you run the for only once, if you filter like this you run a for to remove the unwanted then another for to run on wanted. This way with continue you run only once and skip unwanted.